test/factories/organizations.rb

```diff
@@ -27,7 +27,14 @@
     end
 
     after :create do |orga, evaluator|
-      create_list :hyperlink, evaluator.website_count, linkable: orga
+      evaluator.website_count.times do
+        # create website with faked name, then set it to host 'own' and assign it
+        website = FactoryGirl.create(:website)
+        website.update_columns(host: 'own')
```

Comment:
Ich habe den own-trait durchaus gesehen, habe ihn aber nicht genutzt weil der dann doch immer die gleiche URL generiert (www.example.com), es eine unique validation gibt und in der Factory random 0-3 Webseiten generiert werden sollen.